templates/zerver/integrations/matrix.md

```diff
@@ -0,0 +1,58 @@
+A bridge for exchanging messages between Matrix and Zulip!
+
+{!create-stream.md!}
+
+Next, on your Zulip settings page, create a generic bot for Matrix.
+It is important that you subscribe this bot to the stream which is going
+to act as the bridge.
+Note its username, API key and full name; you will use them in the
+next step. We recommend the name matrix-bot.
+
+In `matrix_bridge_config.py` enter the following details under `zulip`
+keyword:
+```
+"email": "matrix-bot@chat.zulip.org",
+"api_key": "your_key",
+"site": "https://chat.zulip.org",
+"stream": "Stream which acts as the bridge",
+"topic": "Topic of the stream"
+```
+
+Now, create a user on [matrix.org](https://matrix.org/). We recommend a formal
+name similar to 'zulip-bot' for this user.
+
+In `matrix_bride_config.py` enter the follow details under `matrix` keyword:
```

Comment:
This has a typo in the filename.  When writing documentation, it's really important to try accessing all the files you have with copy-paste to make sure they all work; otherwise it's almost guaranteed that you'll have some typos in your documentation (Which has a big impact on how polished the docs feel to users).  

For the IRC piece, I think we might want to have that be a separate page for IRC, at least in the index.

Reply:
@timabbott Are you expecting a list of different options for IRC, ie. matrix and documentation for the existing mirror?

Reply:
No, I just think that something that's an "IRC bridge" thing should have an IRC logo/title in the /integrations index and a heading and so on to make it feel like it's what you want for IRC.

Reply:
Should we have a link at the IRC page which points to the Matrix page?

Reply:
I think it can link to it, but my hope would be that the page is self-contained and one doesn't need to look at the Matrix page to install the IRC bridge successfully.